File: pophory-domain/src/main/java/com/pophory/pophorydomain/albumtheme/infrastructure/AlbumCoverJpaRepository.java
package com.pophory.pophorydomain.albumtheme.infrastructure;

import com.pophory.pophorydomain.albumtheme.AlbumCover;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AlbumCoverJpaRepository extends JpaRepository<AlbumCover, Long> {
}

File: pophory-domain/src/main/kotlin/com/pophory/pophorydomain/albumtheme/infrastructure/AlbumCoverJpaRepository.kt
package com.pophory.pophorydomain.albumtheme.infrastructure

import com.pophory.pophorydomain.albumtheme.AlbumCover
import org.springframework.data.jpa.repository.JpaRepository

interface AlbumCoverJpaRepository : JpaRepository<AlbumCover, Long> {
}